util/os/windows.zig

```diff
@@ -0,0 +1,54 @@
+const std = @import("std");
+const log = std.log.scoped(.zbench_platform_windows);
+
+pub fn getCpuName(allocator: std.mem.Allocator) ![]const u8 {
+    const stdout = try exec(allocator, &.{ "wmic", "cpu", "get", "name" });
+    return stdout[45 .. stdout.len - 7];
+}
+
+pub fn getCpuCores(allocator: std.mem.Allocator) !u32 {
+    // Use `NumberOfLogicalProcessors` to get the logical cores count
+    const stdout = try exec(allocator, &.{ "wmic", "cpu", "get", "NumberOfCores" });
```

Comment:
changed to "numberofcores" == physical cores

Reply:
Thanks for testing it.